ostelco-ios-client/SupportingFiles/Production.plist

```diff
@@ -28,7 +28,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>16</string>
+	<string>21</string>
 	<key>FacebookAppID</key>
 	<string>2328394300810303</string>
```

Comment:
Is this number in any way something we should keep secret or is this public information?

Reply:
It's publicly viewable.